lib/utils.js

```diff
@@ -194,3 +195,21 @@ function getRestifyClientOptions(opts) {
 
 	return (newOpts);
 }
+
+/*
+ * Convert the top level key names to lower case.
```

Comment:
Please add a comment/note that this is using in-place modification (i.e. does not return a new object), my actual preference would be to return a new object - currently this function is returning an object, even though it's the same object (just modified in place), which can lead to confusion (i.e. use one or the other - not both).

Reply:
So like this?

```
function lowerCaseAllKeys(o) {
        var keys = Object.keys(o);
        var new = {};
        keys.forEach(function (key) {
                var l_key = key.toLowerCase();
                new[l_key] = o[key];
        });
        return (new);
}
```

Reply:
Exactly - that also takes care of the ordering issue (i.e. the ordering will be kept the same).